CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## Fixed
+- [vtex test e2e] Screenshot and video assets updated according to tester-hub.
```

Comment:
Didn't quite understand - how about something like:
```
Show screenshot and videos assets URLs received from `vtex.tester-hub` new API.
```
Is this correct?

Reply:
yes, changed it